src/main/collectors/SearchResultCollector.js

```diff
@@ -13,6 +15,7 @@ class SearchResultCollector {
    * @param {function} countResolver - Function that should return the numnber of results in the search
    */
   constructor(phraseResolver, countResolver) {
+    super("search");
```

Comment:
`InstantSearchQueryCollector` uses the same `type`. Is this OK ?

Reply:
It's ok, they are both sending search events